ios/FluentUI/Core/SwiftUI+ViewModifiers.swift

```diff
@@ -66,7 +66,7 @@ extension View {
     ///   - value: The value to check against when determining whether to run the closure.
     ///   - action: A closure to run when the value changes.
     /// - Returns: A view that fires an action when the specified value changes.
-    func onChange_iOS17<V>(of value: V, _ action: @escaping (V) -> Void) -> some View where V: Equatable {
+    public func onChange_iOS17<V>(of value: V, _ action: @escaping (V) -> Void) -> some View where V: Equatable {
```

Comment:
Sorry, but this should not be made public. This is not a feature of Fluent, and we should not be polluting `View` downstream with this change, regardless of convenience to any one client.